joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -678,3 +724,169 @@ TEST_F(TestTrajectoryController, test_partial_joint_list) {
 
   executor.cancel();
 }
+
+/**
+ * @brief invalid_message Test missmatched joint and reference vector sizes
```

Comment:
```suggestion
 * @brief invalid_message Test mismatched joint and reference vector sizes
```

let's save this extra `s` for a rainy day